Changelog.md

```diff
@@ -8,6 +8,7 @@
 - Update Camel Quarkus Catalog from 1.8.1 to 2.0.0
 - Adapt for Camel K community 1.5/Camel K Red Hat 1.4
   - Provide completion for new Camel K modeline option names
+  - Provide completion for the different kind of Camel K modeline option `config` (`configmap`, `secret`, `file`) 

```

Comment:
proposals from @lhein:
"Provide completion for the different kinds of config Camel K modeline option ..."

Reply:
which is better. I will update

Reply:
yeah, but it needs to be "kinds" because there are actually 3 of them.